cli/kata-check_ppc64le.go

```diff
@@ -75,3 +93,14 @@ func archKernelParamHandler(onVMM bool, fields logrus.Fields, msg string) bool {
 func getCPUDetails() (vendor, model string, err error) {
 	return genericGetCPUDetails()
 }
+
+func isSMTOff() bool {
+	// Check if the SMT is off
+	out, err := exec.Command(ppc64CpuCmd, smtStatusOption).Output()
```

Comment:
@nitkon Is it possible to get this through sysfs rather than execing a command here?

Reply:
@amshinde : There is no easy way of getting a conclusive evidence from sysfs. The ppc64_cpu command interprets many files in sysfs to come to this conclusion. So it's better to exec this command as well.

Reply:
I'm really not comfortable with this approach, but if there really is no other way, I guess we have no other alternative :disappointed:

Reply:
I just remembered `ppc64_cpu` also has logic to understand the `subcores_per_core` mode the host is in, which is an additional complication you'd have to deal with. So shelling out to it is probably the least-worst option.

Reply:
Understood. btw, can you run `ppc64_cpu --smt` as a non-`root` user?

Reply:
@jodh-intel : Yes you can run.